src/VirtoCommerce.InventoryModule.Data/ExportImport/InventoryExportImport.cs

```diff
@@ -40,13 +40,15 @@ private int BatchSize
             }
         }
 
-        public InventoryExportImport(IInventoryService inventoryService, IFulfillmentCenterSearchService fulfillmentCenterSearchService,
-            IInventorySearchService inventorySearchService, IFulfillmentCenterService fulfillmentCenterService,
+        public InventoryExportImport(IInventoryService inventoryService,
+            ISearchService<FulfillmentCenterSearchCriteria, FulfillmentCenterSearchResult, FulfillmentCenter> fulfillmentCenterSearchService,
+            IInventorySearchService inventorySearchService,
+            ICrudService<FulfillmentCenter> fulfillmentCenterService,
```

Comment:
arrange params: first generic services, then inventory services, also use the same order in constructor body please